File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for(int i = 0; i < nums.length; i++) {
            for(int j = i; j< nums.length; j++) {
                if((nums[i] + nums[j] == target) && (i != j)) {
                    int[] answer = {i , j};
                    return answer;
                }
            }
        }
        return null;
    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        res = []
        numSet = {}
        for i in range(len(nums)):
            if nums[i] not in numSet: 
                numSet[nums[i]] = 0
            numSet[nums[i]] += 1
            if numSet.get(nums[i]) > (len(nums) / 3): 
                if nums[i] not in res:
                    res.append(nums[i])
        return res